Packs/FiltersAndTransformers/Scripts/RemoveMatches/RemoveMatches.yml

```
@@ -0,0 +1,47 @@
+args:
+- default: true
+  description: The value on which the transformer is applied.
+  isArray: true
+  name: value
+- auto: PREDEFINED
+  defaultValue: "Yes"
+  description: Whether to ignore the case of the item for which you are searching. Default is "Yes".
+  name: ignorecase
+  predefined:
+  - "yes"
+  - "no"
+- auto: PREDEFINED
+  defaultValue: "No"
+  description: Whether to match the exact item in the list, or look for any string that contains it. Default is "No".
+  name: matchexact
```

```suggestion
  name: match_exact
```
Please update accordingly the python file and the readme.